### PR TITLE
documenting test run customization options

### PR DESCRIPTION
**Pull request by mw-hrastega**

### CODE REVIEW

File: CONFIGDOC.md [hunk 1]
```diff
@@ -82,28 +83,38 @@ By default, the **Run MATLAB Tests** task creates a test suite from all the test
...
+![customize_test_run](https://user-images.githubusercontent.com/48831250/217962218-7a47fa54-6fdc-484a-8ce9-e803815afcd2.png)
 
 #### Generate Test and Coverage Artifacts
 To generate test and coverage artifacts, select check boxes in the **Generate test artifacts** and **Generate coverage artifacts** sections. By default, the plugin assigns to each specified artifact a path relative to the working directory. However, you can override the default values:
 
 * If you select a test artifact check box, you can specify the path to store the artifact in the **File path** box.
 * If you select a coverage artifact check box, you can specify the path to store the artifact in the **Folder path** box.
 
-For example, run your tests, and generate a JUnit-style test results report and an HTML code coverage report at the specified locations in your working directory.
+For example, run your tests, and generate test results in JUnit XML format and a code coverage report in HTML format at the specified locations in your working directory.
```
Comment (ppandian28) [inline CONFIGDOC.md:104]: Like the change of removing 'report' for JUnit test result. I feel the current wordings more appropriately reflect on what's being produced. Nice work, Houman!

**Review by ppandian28 — APPROVED** [review]
Changes look good to me.

File: CONFIGDOC.md (hunk 1 above)
Comment (mw-hrastega) [reply CONFIGDOC.md:104]: Thanks Preethi :)

**Review by davidbuzinski — APPROVED** [review]
:shipit:

**Review by mcafaro — APPROVED** [review]
:shipit:

**Review by mw-kapilg — APPROVED** [review]

**Pull request merged**